lib/shared/sdk/usbboot/index.js

```diff
@@ -337,7 +337,7 @@ exports.scan = (options) => {
     device.adaptor = exports.name
 
     // We need to open the device in order to access _configDescriptor
-    debug(`Opening device: ${device.name}`)
+    debug(`Opening device: ${idPair}`)
```

Comment:
Looking at the variables above, would the correct-fix here be to change `${device.name}` to `${device.displayName}` ?

Reply:
Yeah, I think that's correct. Fixed!